lib/keycloak/setup_student_account.rb

```diff
@@ -15,6 +15,9 @@ def initialize(keycloak_client = Rails.configuration.keycloak_client)
     def call(actor_id:)
       student = User.find(actor_id)
       create_keycloak_user(student.email, student.name)
+      sso_data = fetch_keycloak_user_data(student.email)
+      sso_id   = sso_data.fetch('id', nil)
```

Comment:
Is this possible that there is no id in keycloak response? What then?
If not possible then we should raise error if there is no id.

Reply:
No, AFAIK the keycloak response always contains the user UUID.

Reply:
So why we have `nil` as default? shouldn't we raise error here if there is no `id` ?